File: infrastructure/src/main/java/com/study/platform/domain/post/infrastructure/FailedPostSyncJpaRepository.java
package com.study.platform.domain.post.infrastructure;

import com.study.platform.domain.post.model.FailedPostSync;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface FailedPostSyncJpaRepository extends JpaRepository<FailedPostSync, UUID> {
}

File: infrastructure/src/main/kotlin/com/study/platform/domain/post/infrastructure/FailedPostSyncJpaRepository.kt
package com.study.platform.domain.post.infrastructure

import com.study.platform.domain.post.model.FailedPostSync
import org.springframework.data.jpa.repository.JpaRepository
import java.util.UUID

interface FailedPostSyncJpaRepository : JpaRepository<FailedPostSync, UUID>
